notes-webapp/src/main/webapp/vue-app/notes/components/NoteBreadcrumb.vue

```diff
@@ -22,7 +22,7 @@
       </div>
     </div>
     <div v-else class="notes-tree-items notes-long-path d-flex align-center">
-      <div class="notes-tree-item long-path-first-item d-flex text-truncate">
+      <div v-if="noteBreadcrumb && noteBreadcrumb[0]" class="notes-tree-item long-path-first-item d-flex text-truncate">
```

Comment:
Change the default value of the prop noteBreadcrumb to an empty array instead of null and make your tests on array.length